Leanback/README.md

```diff
@@ -10,10 +8,10 @@ This sample is a Videos By Google app, designed to run on an Android TV device (
 - Clone this repo:
 
 ```sh
-git clone https://github.com/googlesamples/androidtv-Leanback.git
+git clone https://github.com/android/tv-samples.git
 ```
 
-- Open the project in [Android Studio][studio].
+- Open the Leanback project in [Android Studio][studio].
```

Comment:
As the repo is renamed to 'tv-samples' under Android perhaps refer to it as such to avoid confusion with the Leanback samples?

Reply:
This one is a little funky because there are three projects in one. You can either import all three into Android Studio and specifically configure the one you want ("Leanback" in this case) or you can import the specific project and the configuration happens automatically. To try to clarify, I changed it to "Open the Leanback project within tv-samples in Android Studio." Let me know if that works/

Reply:
FWIW, I am all for renaming this sample to something more descriptive. With the leanback library, leanback showcase, leanback sample, etc, it can be a bit confusing and overwhelming for new developers to TV to understand the best starting point. This is a bigger undertaking obviously but renaming Leanback so it does not look like the library in the group of samples would remove a lot of confusion to the getting started story. WDYT?

Reply:
Yeah, totally agree. I'll spend some time thinking about the best way to do this since it's definitely not obvious from the project names what the purpose of each project is.